core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -413,32 +421,38 @@ int getSpeed( OSMWay way )
                     speed = tInt;
             } else if (highwaySpeed != null)
             {
-                if (!way.hasTag("service"))
-                    speed = highwaySpeed;
-                else
-                    speed = highwaySpeeds.get("living_street");
+                    if (!way.hasTag("service"))
+                        speed = highwaySpeed;
+                    else
+                        speed = highwaySpeeds.get("living_street");
+                }
```

Comment:
E.g. here is just formatting changes which should be avoided. Maybe we should implement an automated code styler. For NetBeans and IntelliJ we have some configs already.